DOCS/man/ao.rst

```diff
@@ -138,11 +138,21 @@ Available audio output drivers are:
         passthrough (even if the device reports it as supported). Use with
         extreme care.
 
-
 ``coreaudio_exclusive`` (macOS only)
     Native macOS audio output driver using direct device access and
     exclusive mode (bypasses the sound server).
 
+``avfoundation`` (macOS only)
+    Native macOS audio output driver using ``AVSampleBufferAudioRenderer``
+    in AVFoundation, which supports `spatial audio
+    <https://support.apple.com/en-us/HT211775>`_.
+
+    .. warning::
+
+        Turning on spatial audio may hang the playback
```

Comment:
just for me, since i didn't look into this yet, where do you turn it on?

Reply:
Here:

<img width="1280" alt="Screenshot 2024-03-26 at 1 29 11 AM" src="https://github.com/mpv-player/mpv/assets/118280419/a2d85b77-e8d0-44b6-b17f-3a976ac3e6a8">

A device that supports spatial audio is needed.